app/components/DataTable/components/DetailDialog/index.tsx

```diff
@@ -89,7 +89,7 @@ const ObjectAttributes = ({
               expandIcon={<ExpandMoreIcon />}
             >
               <Typography variant="subtitle2">
-                {`${idx}: ${mappedValIsArray ? '[' : '{'}`}
+                {`${!!mappedVal?.username ? `${mappedVal?.firstName} ${mappedVal?.lastName}` : idx}: ${mappedValIsArray ? '[' : '{'}`}
```

Comment:
I take it this isn't something we can customize for individual tables?

Reply:
Could we add a second default - if 'name' is present, then display name?

Reply:
Changes pushed